controller/Cargo.toml

```diff
@@ -15,8 +15,14 @@ kube = { version = "0.63.2", default-features = true, features = [ "derive", "ru
 
 snafu = "0.6"
 tokio = { version = "1", features = ["macros", "rt-multi-thread", "time"] }
+
+actix-web = { version = "4.0.0-beta.9", default-features = false }
```

Comment:
nit: I would keep all of the dependencies together without newlines and sorted alphabetically.